mlflow/models/utils.py

```diff
@@ -208,15 +240,24 @@ def _handle_dataframe_input(df):
                     f"but got '{type(input_example)}'",
                 )
             self.data = _handle_dataframe_input(self._inference_data)
-            self.info = {
-                "artifact_path": example_filename,
-                "type": "dataframe",
-                "pandas_orient": "split",
-            }
+            self.info.update(
+                {
+                    "type": "dataframe",
+                    "pandas_orient": "split",
+                }
+            )
```

Comment:
How `artifact_path` was used?

Reply:
never mind.

Reply:
It's used to store the filename that saves input_example. I put that into self.into={...} above.